astropy/table/tests/test_table.py

```diff
@@ -2392,3 +2392,15 @@ class MyTable(table.Table):
         assert type(t['c']) is MyMaskedColumn
         assert type(t['d']) is MySubColumn
         assert type(t['e']) is MySubMaskedColumn  # sub-class not downgraded
+
+
+def test_iterrows():
+    t = table.Table([(2, 3, 2, 1), (8, 7, 6, 5)], names=('a', 'b'))
+    assert t.iterrows(['a']) == zip(t['a'])
```

Comment:
I don't think this test is meaningful as it is just comparing two zip generators.  You need to write out a loop like the doc string example and prove that things are working as expected.